ew/cmd/slimeoid/slimeoid_creation.py

```diff
@@ -141,6 +176,103 @@ async def incubate_slimeoid(cmd):
+        else:
+            response = "Huh? What'd you say?"
+
+    # Check for if the player is at Waffle House
+    elif cmd.message.channel.name != ewcfg.channel_wafflehouse:
+        response = "You can't exactly summon anything in {}. Go to Waffle House first.".format(cmd.message.channel.name)
+
+    # Check if the player already has a Slimeoid or a Negaslimeoid.
+    elif slimeoid_data.life_state == ewcfg.slimeoid_state_active:
+        if slimeoid_data.sltype == ewcfg.sltype_nega:
+            response = "You already have a Negaslimeoid by your side. Destroy your current Negaslimeoid before you commune with the Ancient Ones yet again."
+        else:
+            response = "You wish to summon a Negaslimeoid, yet you have a Slimeoid that still calls you its master. !destroyslimeoid if you wish to commune with the Ancient Ones properly."
+
+    # Check if the player is already incubating a Slimeoid or Negaslimeoid
+    elif slimeoid_data.life_state == ewcfg.slimeoid_state_forming:
+        if slimeoid_data.sltype == ewcfg.sltype_nega:
+            response = "You are already in the process of conjuring a Negaslimeoid."
+        else:
+            response = "You wish to summon a Negaslimeoid, yet you have a Slimeoid that is still yet to call you its master. !destroyslimeoid if you wish to commune with the Ancient Ones properly."
+
+
+    else:
+        
+        #Check if player has too many slimeoids
+        slimeoid_count = get_slimeoid_count(user_id=cmd.message.author.id, server_id=cmd.guild.id)
+        
+        if slimeoid_count >= 3:
+            response = "You have too many slimeoids."
+            # Go to final response
+
+
+        else:
+            #Check if player has a negapoudrin
+            negapoudrin = bknd_item.find_item(item_search=ewcfg.item_id_negapoudrin, id_user=cmd.message.author.id, id_server=cmd.guild.id if cmd.guild is not None else None, item_type_filter=ewcfg.it_item)
+            if negapoudrin is None:
+                response = "You need a negapoudrin to bind a Negaslimeoid to your command."
+                # Go to final response
+    
+
+            else:
+                # Get argument for how big the negaslimeoid should be made
+                sacrificed_negaslime = None
+                if cmd.tokens_count > 1:
+                    sacrificed_negaslime = ewutils.getIntToken(tokens=cmd.tokens, allow_all=True)
+                    # -1 from getIntToken() means use all slime
+                    if sacrificed_negaslime == -1:
+                        sacrificed_negaslime = -user_data.slimes
+
+                # Check if player sacrificed any negaslime or if they have enough to sacrifice.
+                if sacrificed_negaslime == None:
+                    response = "You must sacrifice your own negaslime to conjure a Negaslimeoid. Specify how much negaslime you will sacrifice."
+                    # Go to final response
+
+                elif -sacrificed_negaslime < user_data.slimes:
```

Comment:
if the player puts in a negative number here then from the looks of things it'll give them positive slime. best practice would probably be to invert all numbers if they're <0, since people could put in negative or positive numbers depending on what they think the bot is asking for.